src/services/inlayHints.ts

```diff
@@ -74,6 +74,10 @@ namespace ts.InlayHints {
             return forEachChild(node, visitor);
         }
 
+        function shouldWrapParameterList(node: Node): boolean {
+            return isArrowFunction(node) && node.parameters.length === 1 && !findChildOfKind(node, SyntaxKind.OpenParenToken, file);

```

Comment:
There should be a method like `getFirstChild(file)` on `node` and you can check to see whether its kind is an `OpenParenToken`.

Reply:
Thanks, updated